### PR TITLE
feat(cycle-hmr): adding cycle-hmr ᕙ༼ຈل͜ຈ༽ᕗ

### PR DESCRIPTION
**Pull request by Cmdv**
**Description:**
adding cycle-hmr

**Related issue (if exists):**
you're an issue! 👅

### CODE REVIEW

File: webpack.config.js [hunk 1]
```diff
@@ -5,15 +5,21 @@ const ExtractTextPlugin = require('extract-text-webpack-plugin')
 const srcPath = path.resolve('./src')
 const imagePath = path.resolve('./dist/images')
 
+console.log(process.env.BUILD_ENV);
```
Comment (Cmdv) [inline webpack.config.js:8]: shit need to remove this

File: .babelrc [hunk 2]
```diff
@@ -1,6 +1,9 @@
 {
   "presets": ["es2015", "stage-0"],
   "plugins": [
+    ["cycle-hmr", {
+      "include": "./src/**/**/**/**.js"
```
Comment (Cmdv) [inline .babelrc]: not sure it needs to go this deep

**Comment by Cmdv** [comment]
can't seem to compile the css and then get a weird error on hot-reload:

screen shot 2016-05-18 at 21 25 40

@whitecolor you might have a clue about this? :)

**Comment by wclr** [comment]
hm, no clue, how it can be reproduced?

btw cycle-hmr should be used only while development, not while prod build, so should use `env` in `babelrc`

**Comment by wclr** [comment]
And also if the repo is supposed to use `xstream` it is better to use `cycle-hmr/xstream` as plugin, not to mess with ignoring of missing adapters.

**Comment by Cmdv** [comment]
it is in dev for now as I'm just trying to test it out [here](https://github.com/Cmdv/xstream-boilerplate/blob/cb55b5f9a767088857156bef22a3095d84ecb57d/package.json#L29) 

do you mean import the dep `cycle-hmr/xstream` in the packages ?

**Comment by wclr** [comment]
.babelrc

```
"env": {
  "developement": { // NODE_ENV should be "development"
     "plugins":  ["cycle-hmr/xstream", {...}]
   }
}
```

Will look in to it.

**Comment by Cmdv** [comment]
Cool cheers @whitecolor honestly setting up a project is what takes the longest!!! 😂😂

**Comment by TylorS** [comment]
@Cmdv @whitecolor Thank you for everything in here. I used what was here to create #8 to get it up to date with master :)

**Pull request closed without merging**